GameServer/spells/ResistBuff.cs

```diff
@@ -61,6 +61,14 @@ public class BodyResistBuff : AbstractResistBuff
 
 		// constructor
 		public BodyResistBuff(GameLiving caster, Spell spell, SpellLine line) : base(caster, spell, line) { }
+
+		public override void TooltipDelve(ref MiniDelveWriter dw)
+		{
+			base.TooltipDelve(ref dw);
+			dw.AddKeyValuePair("Function", "resistance");
+			dw.AddKeyValuePair("parm", "16");

```

Comment:
Can this parameter be factorized above under the AbstractResistBuff? Plus this is likely the same for resist debuffs, isn't it?